e2e/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@ultraviolet/e2e",
```

Comment:
can be rename into @repo/e2e ?

Reply:
why is that?

Reply:
https://turbo.build/repo/docs/crafting-your-repository/structuring-a-repository#anatomy-of-a-package

Reply:
> https://turbo.build/repo/docs/crafting-your-repository/structuring-a-repository#anatomy-of-a-package

It's best practice to use a namespace prefix for your [Internal Packages](https://turbo.build/repo/docs/core-concepts/internal-packages) to avoid conflicts with other packages on the npm registry. For example, if your organization is named acme, you might name your packages @acme/package-name.

We use @repo in our docs and examples because it is an unused, unclaimable namespace on the npm registry. You can choose to keep it or use your own prefix.